### PR TITLE
DM-4398 update homepage welcome banner styling

### PR DESCRIPTION
**Pull request by PhilipDeFraties**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-4398

## Description - what does this code do?
updates welcome banner on homepage to be in line with 2024 redesign

## Testing done - how did you test it/steps on how can another person can test it 
verify the layout mirrors the figma desing: https://www.figma.com/file/ccwLbt94U9QGfnuFCwOXzY/Core-Designs---VA-DM?type=design&node-id=10914-1114&mode=design&t=yg4Ih9zR6Izf89cY-0

## Screenshots, Gifs, Videos from application (if applicable)
Screenshot 2024-01-17 at 4 40 15 PM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/assets/stylesheets/dm/pages/_home.scss [hunk 1]
```diff
@@ -4,27 +4,10 @@
...
-  .homepage-welcome-banner-image-container {
-    @include u-position('absolute');
-    bottom: -87px;
-
-    @media screen and (min-width: 1024px) {
-      height: 223px;
-      @include u-right(0);
-      bottom: unset;
-    }
+    background: #162E51;
```
Comment (camillevilla) [inline app/assets/stylesheets/dm/pages/_home.scss:7]: Nice cleanup on this section. This matches the footer color, which uses `.bg-primary-darker`. Let's use the utility class on the HTML element

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -6,27 +6,18 @@
     </div>
   </div>
   <!-- "Welcome" section -->
-  <section class="homepage-welcome-banner dm-gradient-banner-318" role="region" aria-label="Homepage welcome banner">
-    <div class="grid-container height-full">
-      <div class="grid-row height-full flex-align-center">
+  <section class="homepage-welcome-banner usa-section padding-top-3 padding-bottom-7" role="region" aria-label="Homepage welcome banner">
+    <div class="grid-container">
+      <div class="grid-row">
```
Comment (camillevilla) [inline app/views/home/index.html.erb:11]:
```suggestion
      <div class="grid-row text-center text-white flex-justify-center">
```
1. Set the `text` properties up here so you don't have to specify per element
2. use USWDS flexbox properties and you can remove the `grid-col-offset-1` and other margin properties on the `<p>` tag. As much as possible, we want USWDS and the automagic of flexbox to do the heavy lifting on calculating margins for us.

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -6,27 +6,18 @@
...
         <div class="grid-col-12">
-          <h1 class="usa-prose-h1 text-white margin-top-4 margin-bottom-4 desktop:margin-top-0">
+          <h1 class="usa-prose-h1 text-white">
```
Comment (camillevilla) [inline app/views/home/index.html.erb:13]:
```suggestion
          <h1 class="usa-prose-h1">
```

File: app/views/home/index.html.erb [hunk 4]
```diff
@@ -6,27 +6,18 @@
...
             Discover VA innovations to adopt at your facility
           </h1>
-          <div class="grid-row position-relative">
-            <div class="homepage-welcome-banner-body-text-container grid-col-12 desktop:grid-col-7 padding-right-105">
-              <p class="usa-prose-body text-white margin-bottom-2">
+          <div class="grid-row">
```
Comment (camillevilla) [inline app/views/home/index.html.erb:16]: remove this extra `grid-row` and make `homepage-welcome-banner` a sibling of the header

File: app/views/home/index.html.erb [hunk 5]
```diff
@@ -6,27 +6,18 @@
...
+            <div class="homepage-welcome-banner-body-text-container grid-col-9 grid-offset-1 ">
+              <p class="usa-prose-body text-white margin-top-1 text-center desktop:margin-x-0 mobile:margin-x-2 tablet:margin-x-4">
```
Comment (camillevilla) [inline app/views/home/index.html.erb:18]: See comment above
```suggestion
              <p class="usa-prose-body margin-top-1">
```

File: app/views/home/index.html.erb [hunk 6]
```diff
@@ -6,27 +6,18 @@
...
         <div class="grid-col-12">
-          <h1 class="usa-prose-h1 text-white margin-top-4 margin-bottom-4 desktop:margin-top-0">
+          <h1 class="usa-prose-h1 text-white">
             Discover VA innovations to adopt at your facility
           </h1>
-          <div class="grid-row position-relative">
-            <div class="homepage-welcome-banner-body-text-container grid-col-12 desktop:grid-col-7 padding-right-105">
-              <p class="usa-prose-body text-white margin-bottom-2">
+          <div class="grid-row">
+            <div class="homepage-welcome-banner-body-text-container grid-col-9 grid-offset-1 ">
```
Comment (camillevilla) [inline app/views/home/index.html.erb:17]:
```suggestion
            <div class="homepage-welcome-banner-body-text-container grid-col-12 desktop:grid-col-9">
```
See screenshot in parent comment. Let's collapse the side margins on this text when we're in mobile and tablet widths. Note that USWDS assumes mobile is the default width (`grid-col-12`). You can use `desktop:grid-col-9` to define the desire behavior for desktop.

**Review by camillevilla — CHANGES_REQUESTED** [review]
Good first pass on this, @PhilipDeFraties. I appreciate your diligence on cleaning up SCSS for removed components. Take a closer look at the mobile designs in Figma for some margin differences we want to address:
Screenshot_2024-01-18_at_3_19_29_PM

In case it's hard to grok all the suggested changes via comments, here's the markup snippet I ended up with to address the side margins:
```html
<section class="homepage-welcome-banner usa-section padding-top-3 padding-bottom-7" role="region" aria-label="Homepage welcome banner">
    <div class="grid-container">
      <div class="grid-row flex-justify-center text-white text-center">
          <h1 class="grid-col-12 usa-prose-h1">
            Discover VA innovations to adopt at your facility
          </h1>
        <div class="homepage-welcome-banner-body-text-container grid-col-12 desktop:grid-col-9">
            <p class="usa-prose-body margin-top-1">
              We’re a discovery and collaboration tool that curates VA’s promising innovations, encourages their diffusion, and fosters engagement with greater healthcare communities.
            </p>
          </div>
      </div>
    </div>
  </section>
 ```
Screenshot 2024-01-18 at 3 57 15 PM

Some general tips:
1. As much as possible, try to avoid nesting `grid-col-x` divs within other `grid-col-x` divs. 
2. if you find yourself struggling with margins, play around more with some flexbox utilities so we can avoid calculating whitespace. Try to avoid `grid-offset-x` moving forward. 

After we're done, let's also pass this over to Nick for some design review.

File: app/views/home/index.html.erb [hunk 7]
```diff
@@ -6,35 +6,22 @@
     </div>
   </div>
   <!-- "Welcome" section -->
-  <section class="homepage-welcome-banner dm-gradient-banner-318" role="region" aria-label="Homepage welcome banner">
-    <div class="grid-container height-full">
-      <div class="grid-row height-full flex-align-center">
-        <div class="grid-col-12">
-          <h1 class="usa-prose-h1 text-white margin-top-4 margin-bottom-4 desktop:margin-top-0">
+  <section class="usa-section padding-top-3 padding-bottom-7 bg-primary-darker" role="region" aria-label="Homepage welcome banner">
```
Comment (PhilipDeFraties) [inline app/views/home/index.html.erb:9]: @camillevilla just went ahead and removed the `homepage-welcome-banner` css classes

File: app/views/home/index.html.erb [hunk 8]
```diff
@@ -6,35 +6,22 @@
...
-            <div class="homepage-welcome-banner-image-container text-white grid-col-12 tablet:grid-col-6 desktop:grid-col-5 desktop:padding-left-105">
-              <%= image_tag(
-                    'homepage-image.png',
-                    class: 'display-block padding-left-0',
-                    alt: 'VA Secretary McDonough speaks with a clinician during his visit to the D.C. VA Medical Center in February 2021.'
-                  )
-              %>
-            </div>
+        <div class="grid-col-12 desktop:grid-col-9">
+            <p class="usa-prose-body margin-top-1">
```
Comment (camillevilla) [inline app/views/home/index.html.erb:16]:
```suggestion
            <p class="usa-prose-body margin-top-1 text-center">
```

**Review by camillevilla — CHANGES_REQUESTED** [review]
One more bit of markup to get alignment right in responsive: 
### Before
Screenshot 2024-01-23 at 2 23 15 PM

### After
Screenshot 2024-01-23 at 2 23 06 PM

**Review by camillevilla — APPROVED** [review]
✅ Looking good! Let's get this out to DEV for one more visual design review from Nick.

**Pull request merged**